src/OrganisationRegistry.Api/Backoffice/Admin/Configuration/Queries/ConfigurationListQuery.cs

```diff
@@ -26,7 +26,7 @@ protected override IQueryable<ConfigurationValue> Filter(FilteringHeader<Configu
             if (!filtering.ShouldFilter)
```

Comment:
Should we use the same approach here as in https://github.com/Informatievlaanderen/organisation-registry/pull/526/files#diff-0d8c1a3d7a57515acf5d966b4727f53e436b16de6f0c189ff03d8c728a975be8R104 ?